src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticIds.cs

```diff
@@ -49,5 +49,7 @@ internal static class RoslynDiagnosticIds
         public const string DoNotCallGetTestAccessorRuleId = "RS0043";
         public const string CreateTestAccessorRuleId = "RS0044";
         public const string ExposeMemberForTestingRuleId = "RS0045";
+
+        public const string WrapStatementsRuleId = "RS0046";
```

Comment:
Given these are temporary rules, lets start the IDs from a fresh range.

```suggestion
        public const string WrapStatementsRuleId = "RS0100";
```

Reply:
sure!